src/storage.rs

```diff
@@ -299,6 +299,7 @@ pub fn read_file_line_by_line(
                 // max line length of 1MiB
                 LinesCodec::new_with_max_length(1024 * 1024),
```

Comment:
Not using the LineTaker yet?

Reply:
it is used as we discussed, also renamed to `TakeFromIterable`